<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSSSAOWindow</class>
 <widget class="QWidget" name="SSSSAOWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSAO window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QPushButton" name="qUndockButton">
     <property name="text">
      <string>Undock</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="qGLFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="cameraType_combo"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>FPS camera speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="fpsCamSpeedSpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>ZNear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="znearSpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>ZFar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="zfarSpinBox">
         <property name="maximum">
          <double>50000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Render result</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="renderResultComboBox"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
